src/libraries/Microsoft.Extensions.Caching.Abstractions/ref/Microsoft.Extensions.Caching.Abstractions.net7.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I would rename this file to `Microsoft.Extensions.Caching.Abstractions.net7.0.cs` as we usually use one the following suffixes whenever we need to condition on target frameworks:
- TargetFrameworkIdentifier: either `netcoreapp`, `netstandard` or `netframework`
- TargetFramework: i.e. `netstandard2.0`, `net462`, `net6.0`, `net7.0`, ...